src/main.cpp

```diff
@@ -599,7 +603,7 @@ static void init_log_streams(const Settings &cmd_args)
 		warningstream << "Deprecated use of debug_log_level with an "
 			"integer value; please update your configuration." << std::endl;
 		static const char *lev_name[] =
-			{"", "error", "action", "info", "verbose"};
+			{"", "error", "action", "info", "verbose", "trace"};
```

Comment:
How about documenting this value in `settingtypes.txt`? Since it's a valid option, it would make sense to expose that too, as an alternative to `--trace`.

Reply:
Done, also added to minetest.conf.example.